pkg/kubectl/polymorphichelpers/canbeautoscaled_test.go

```diff
@@ -17,51 +17,105 @@ limitations under the License.
 package polymorphichelpers
 
 import (
+	"fmt"
 	"testing"
 
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	extensionsv1beta1 "k8s.io/api/extensions/v1beta1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/cli-runtime/pkg/genericclioptions/resource"
+	scalefake "k8s.io/client-go/scale/fake"
+	testing2 "k8s.io/client-go/testing"
+	"k8s.io/kubernetes/pkg/kubectl"
 )
 
-func TestCanBeAutoscaled(t *testing.T) {
+func TestCoreTypesCanBeAutoscaled(t *testing.T) {
```

Comment:
why do we need to change the function name here?

Reply:
My doubt was that test case name  according to Golang conventions should be TestFunctionName